### ISSUE TITLE
v3.2.0 org.genivi.commonapi.core  compile fails

### ISSUE THREAD

**Issue by phongt** [GENIVI/capicxx-core-tools#25]
Have just checked out the v3.2.0 release and got the compiler failure

```
[ERROR] 	import org.genivi.commonapi.core.Deployment;
[ERROR] 	       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[ERROR] The import org.genivi.commonapi.core.Deployment cannot be resolved
```

Build Environment:
* Ubuntu 20.04.1 LTS 
* Eclipse DSL Tools
** Version: 2018-09 (4.9.0)
** Build id: 20180917-1800
* java-8-openjdk

[build.log](https://github.com/GENIVI/capicxx-core-tools/files/5470163/build.log)

**Comment by rlokanathan** [comment]
I have the same error too. I am using Windows environment to build the generators. And my java version is java1.8

**Comment by rchelim** [comment]
Same error here, with Ubuntu 20.04 and java1.8.

**Comment by ahmdsamy** [comment]
Same error, ubuntu 18.04 and java 1.8

**Comment by phongt** [comment]
@lutzbichler

**Comment by lutzbichler** [comment]
The generated code is not included in v3.2.0. Added it and released v3.2.0.1.

**Comment by DoNotSmoking** [comment]
> The generated code is not included in v3.2.0. Added it and released v3.2.0.1.
@lutzbichler  can you provide the  way to generate the missed code？ thanks.
